numba/tests/npyufunc/test_gufunc.py

```diff
@@ -140,6 +162,24 @@ def sum_row(inp, out):
         with self.assertRaisesRegex(TypeError, msg):
             sum_row(inp)
 
+    def test_axis(self):
+        @guvectorize("(n)->(n)")
+        def my_cumsum(x, res):
+            acc = 0
```

Comment:
Same comment as above.

Reply:
done in https://github.com/numba/numba/pull/6786/commits/20c272d5d387c477197cd3bee18abc6adc445a32